pkg/parser/parser.go

```diff
@@ -24,10 +32,44 @@ func executeBootstrapTemplate() (string, error) {
 // this function will eventually take a pointer to the bootstrap contract struct.
 // it will then template out the variables into the final bootstrap trigger script.
 func Parse() {
-	triggerBootstrapScript, err := executeBootstrapTemplate()
+	inputJSON, err := json.Marshal(getBaseTemplate())
+	if err != nil {
+		log.Printf("Failed to marshal the nbcontractv1 to json: %v", err)
+	}
+
+	log.Println("Input Json: ")
+	log.Println(string(inputJSON))
+
+	// inputJson above will be provided by bootstrappers. We are using getBaseTemplate() for dev/test purpose for now.
+	// We can further move it to other file for unit tests later.
+
+	inputContract := nbcontractv1.Configuration{}
+	err = json.Unmarshal(inputJSON, &inputContract)
+	if err != nil {
+		log.Printf("Failed to unmarshal the json to nbcontractv1: %v", err)
+	}
+
+	triggerBootstrapScript, err := executeBootstrapTemplate(&inputContract)
 	if err != nil {
 		log.Printf("Failed to execute the template: %v", err)
 	}
 
+	log.Println("output env vars:")
 	log.Println(triggerBootstrapScript)
 }
+
+func getBoolStr(state nbcontractv1.FeatureState) string {
+	if state == nbcontractv1.FeatureState_FEATURE_STATE_ENABLED {
+		return "true"
+	}
+
+	return "false"
+}
+
+func getInverseBoolStr(state nbcontractv1.FeatureState) string {
```

Comment:
Some env vars need inverse bool str, for example, disable_ssh. Let me know if there is other way to do this better.

Reply:
hmm i am not fond of this but we can iterate on this going forward, wont block the PR on this.